src/settings/Settings.js

```diff
@@ -180,7 +180,7 @@ export const SETTINGS = {
     "fontSize": {
```

Comment:
so given that we aren't changing the settings key, what effect will this have if someone changed their font already?

Reply:
We are changing the settings key. That's why I took it off of the review queue